cloud_auto.yaml

```diff
@@ -104,3 +139,6 @@ Outputs:
   WebsiteURL:
     Description: URL for website
     Value: !Join ['', ['http://', !GetAtt [MyLoadBalancer, DNSName]]]
+  ELBAccessLogsBucket:
+    Description: DNS name of the S3 bucket storing ELB Access Logs
+    Value: !GetAtt [S3LoggingBucket, DomainName]
```

Comment:
Lets use single style. You can use instead
`!GetAtt S3LoggingBucket.DomainName`

Reply:
fixed